prometheus/README.md

```diff
@@ -1,12 +1,24 @@
 # Setting up Prometheus
 
-## Installation
+## Installation (RHEL/CENTOS 7)
 
-* Install latest Prometheus package from Crunchy Repository
-* Install latest Alertmanager package from Crunchy Repository
-* Install latest pgmonitor-prometheus-extras package
-* Install latest pgmonitor-alertmanager-extras package
+RPMs are available to Crunchy customers through our access portal. The RPM will take care of all the steps below in the Installation section. From the Setup section onward, the instructions are relevant no matter the installation method.
```

Comment:
There are currently RPM packages available to [Crunchy Data](https://www.crunchydata.com) customers through the Crunchy Data [access portal](https://access.crunchydata.com/).  Installing the RPM will take care of all of the steps described in the installation section below.  If you install via the RPM, you can continue reading at the [Setup](#setup) section.